### PR TITLE
Updated test cases pt2

### PR DESCRIPTION
**Pull request by ivodimitrov**
Completed tasks:
1. Scroll to item method
2. Updated `Settings` screen
3. Created `Accounts` screen
4. Created `Email in` screen
5. Created `Help & support` screen
6. Updated Settings suite test
image

### CODE REVIEW

**Comment by ivodimitrov** [comment]
[Scroll to item method](https://trello.com/c/vn8rDwbQ/189-scroll-to-item-method)

**Comment by ivodimitrov** [comment]
[New Settings screen](https://trello.com/c/geVwGMno/185-new-settings-screen)

**Comment by ivodimitrov** [comment]
[New Settings screen](https://trello.com/c/geVwGMno/185-new-settings-screen)

**Comment by ivodimitrov** [comment]
[iOS update OpenSourceLicensesScreen & C16181](https://trello.com/c/golTFqcE/181-ios-update-opensourcelicensesscreen-c16181)

**Comment by ivodimitrov** [comment]
[iOS updated test cases](https://trello.com/c/vjtnQO09/174-ios-updated-test-cases)

**Comment by ivodimitrov** [comment]
[Update Settings suite test cases pt.1](https://trello.com/c/x8SAw1bG/190-update-settings-suite-test-cases-pt1)

**Comment by ivodimitrov** [comment]
[Update Settings suite test cases pt.2](https://trello.com/c/DxJmjmUY/195-update-settings-suite-test-cases-pt2)

File: lib/ios/screens/settings/accounts_screen.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  # This method will find user by given email and swipe left on it.
+  def swipe_left_on_user(email)
+    element = find_user email
+    execute_script 'mobile: swipe', direction: 'left', element: element.ref
+  end
+
+  # This method will get value of current login username.
+  def current_login_username_value
+    driver.find_element(:xpath, '//XCUIElementTypeTable/XCUIElementTypeCell[1]/XCUIElementTypeStaticText[1]').attribute('value')
```
Comment (ivarbanovv) [inline lib/ios/screens/settings/accounts_screen.rb:47]: Xpath shorter ?

File: lib/ios/screens/settings/open_source_licenses_screen.rb [hunk 2]
```diff
@@ -15,5 +15,16 @@ class OpenSourceLicensesScreen < Base
...
   text(:sv_progress_hud_text, 'accessibility_id~SVProgressHUD')
   text(:segment_analytics_for_ios_text, 'accessibility_id~Segment Analytics for iOS')
+  text(:plaid_link_for_ios_text, 'accessibility_id~Plaid Link for iOS')
+  text(:zendesk_text, 'accessibility_id~Zendesk')
+  text(:peasy_text, 'accessibility_id~Peasy')
+  text(:pixel_test_text, 'accessibility_id~PixelTest')
+  text(:tab_test_kit_text, 'accessibility_id~TABTestKit')
+  text(:lottie_text, 'accessibility_id~Lottie')
+
+  # This method will scroll to SQLite swift element.
```
Comment (ivarbanovv) [inline lib/ios/screens/settings/open_source_licenses_screen.rb:25]: need more context about this element and why we need a special method for it

File: spec/ios/settings/settings_spec.rb [hunk 3]
```diff
@@ -91,11 +99,20 @@
     expect(on(OpenSourceLicensesScreen).rx_data_sources_text?).to be true
     expect(on(OpenSourceLicensesScreen).apollo_for_ios_text?).to be true
     expect(on(OpenSourceLicensesScreen).kingfisher_text?).to be true
-    on(OpenSourceLicensesScreen).scroll
+    on(OpenSourceLicensesScreen).scroll_to_sqlite_swift_text
```
Comment (ivarbanovv) [inline spec/ios/settings/settings_spec.rb:102]: scroll_to_sqlite.... method name does not describe a proper test action

File: spec/ios/settings/settings_spec.rb [hunk 4]
```diff
@@ -212,3 +274,50 @@
...
+    on(TopNavigation).settings_button
+  end
+
+  it 'sees correct "BK logo" and "BK Company Name" on Settings screen', :tms => 'C16733' do
+    expect(on(SettingsScreen).logo_image_enabled?).to be true
+    expect(on(SettingsScreen).text_visible_onscreen? @company_name).to be true
+    expect(on(SettingsScreen).text_visible_onscreen? @company_email).to be true
+  end
+
+  it 'turns on "open on camera (Partner account)" on Settings screen', :tms => 'C11926' do
```
Comment (ivarbanovv) [inline spec/ios/settings/settings_spec.rb:299]: block way too long

File: lib/ios/screens/settings/accounts_screen.rb (hunk 1 above)
Comment (ivodimitrov) [reply lib/ios/screens/settings/accounts_screen.rb:47]: Unfortunately cannot be shorter :(

File: lib/ios/screens/settings/open_source_licenses_screen.rb (hunk 2 above)
Comment (ivodimitrov) [reply lib/ios/screens/settings/open_source_licenses_screen.rb:25]: Removed method and elements as we agreed, that they are not important.

File: spec/ios/settings/settings_spec.rb (hunk 3 above)
Comment (ivodimitrov) [reply spec/ios/settings/settings_spec.rb:102]: Removed method and shortened the test. Now test verifies the screen by header and one element.

File: spec/ios/settings/settings_spec.rb (hunk 4 above)
Comment (ivodimitrov) [reply spec/ios/settings/settings_spec.rb:299]: Separated as two independent tests.

**Review by ivarbanovv — APPROVED** [review]
merge it

**Pull request merged**